action.yml

```
@@ -52,7 +52,7 @@ runs:
   using: 'composite'
   steps:
     - name: Install jq
-      uses: dcarbone/install-jq-action@8867ddb4788346d7c22b72ea2e2ffe4d514c7bcb
+      uses: dcarbone/install-jq-action@e397bd87438d72198f81efd21f876461183d383a
```

Adding a comment to make the commit more readable
```suggestion
      uses: dcarbone/install-jq-action@e397bd87438d72198f81efd21f876461183d383a # v3.0.1
```